src/app/companies/page.tsx

```diff
@@ -1,12 +1,32 @@
 import HeaderLayout from "~/components/header-layout";
+import { RoleReviewCard } from "~/components/role-review-card";
 import SearchFilter from "~/components/search-filter";
+import { api } from "~/trpc/server";
+import { unstable_noStore as noStore } from "next/cache";
 
 export default async function Companies() {
+  /**
+   * FIXME: This is a temporary fix, figure out how to get build command working without noStore();
+   * @returns A promise containing the roles from the database
+   */
```

Comment:
One possible fix is to make this a client component and copy the structure of the roles/page.tsx to avoid this noStore issue. I'm definitely seeing (or imagining) some performance issues for this page